apps/www/registry/default/example/cards/chat.tsx

```diff
@@ -142,24 +144,30 @@ export function CardsChat() {
           <form
             onSubmit={(event) => {
               event.preventDefault()
-              setMessages([
-                ...messages,
-                {
-                  role: "user",
-                  content: event.currentTarget.message.value,
-                },
-              ])
-
-              event.currentTarget.message.value = ""
+              if (messageLength > 0) {
```

Comment:
Instead of using a State to maintain and check for message length, try using "event.currentTarget.message.value" directly to check message length. something like this:-

```js
const messageInput = event.currentTarget.message.value;

if(messageInput?.length > 0){
  // Add Message
}

/// or just return from function if message is empty
if(messageInput?.length < 1){
  return
}

```

Reply:
Hi @SRX9, I am using state to track the message length in order to disable/enable the send button on the form which is seen in most chat apps today.

Using `event.currentTarget.message.value` works also, it does mean that the messageInput type is `any`. And then you continue to use `trim()` and `.length` on something of type any which shows no problems on the surface. Changing the id of the input will also easily go unnoticed.

Both options work, but the state would still be there to disable/enable send button when input length is < 1.

Reply:
Ok, Then its fine